src/js/lib.js

```diff
@@ -304,7 +301,7 @@ var isEmpty = function(obj) {
  * @private
  */
 var hasClass = function(element, classToCheck){
-  return ((' ' + element.className + ' ').indexOf(' ' + classToCheck + ' ') !== -1);
+  return ((` ${element.className} `).indexOf(` ${classToCheck} `) !== -1);
```

Comment:
With the template syntax you can clearly see there's a space inside of the template, so I'm not sure I follow why one infers reasoning behind that space more than the other.